legate/tester/stages/util.py

```diff
@@ -123,11 +123,15 @@ def log_proc(
     """Log a process result according to the current configuration"""
     if config.debug or config.dry_run:
         LOG(shell(proc.invocation))
-    duration = (
-        f" {{{proc.time.total_seconds():0.2f}s}}"
-        if proc.time is not None
-        else ""
-    )
+
+    if proc.time is None or proc.start is None or proc.end is None:
+        duration = ""
+    else:
+        time = yellow(f"{proc.time.total_seconds():0.2f}s")
+        start = f"{proc.start.strftime('%H:%M:%S.%f')[:-4]}"
+        end = f"{proc.end.strftime('%H:%M:%S.%f')[:-4]}"
```

Comment:
```suggestion
        time = f"{proc.time.total_seconds():0.2f}s"
        start = proc.start.strftime("%H:%M:%S.%f")[:-4]
        end = proc.end.strftime("%H:%M:%S.%f")[:-4]
```

Reply:
I mean I guess you could not-use f-strings who knew that was possible :rofl: